### PR TITLE
git-root: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

**Comment by CleanMachine1** [comment]
The extra commits seemed to have been merged from git-extras. I may have misclicked and started a branch in VSCode from git-effort

**Comment by bl-ue** [comment]
@CleanMachine1 once #6002 is merged that won't be a problem.

File: pages/common/git-root.md
Comment (marchersimon) [inline pages/common/git-root.md:3]:
```suggestion
> Print the current working directory or Git repository.
```

File: pages/common/git-root.md
Comment (marchersimon) [inline pages/common/git-root.md:7]:
```suggestion
- Print the absolute path of the current Git repository:
```

File: pages/common/git-root.md
Comment (marchersimon) [inline pages/common/git-root.md:11]:
```suggestion
- Print the current working directory, relative to the root of the current Git repository:
```

File: pages/common/git-root.md
Comment (CleanMachine1) [reply pages/common/git-root.md:3]:
```suggestion
> Print the current working directory or Git repository directory.
```

File: pages/common/git-root.md
Comment (CleanMachine1) [reply pages/common/git-root.md:11]: This is a little man pagey

File: pages/common/git-root.md
Comment (marchersimon) [reply pages/common/git-root.md:3]: `Git repository directory` doesn't really sound right. Maybe `Git repository path`?

File: pages/common/git-root.md
Comment (marchersimon) [reply pages/common/git-root.md:11]: I don't think that has to be a bad thing.

File: pages/common/git-root.md
Comment (CleanMachine1) [reply pages/common/git-root.md:3]:
```suggestion
> Print the current working directory or Git repository path.
```

Comment (navarroaxel) [reply pages/common/git-root.md:3]:
```suggestion
> Print the root directory of the current Git repository.
```

File: pages/common/git-root.md
Comment (bl-ue) [reply pages/common/git-root.md:11]: Neither do I.  You know, "the" is used in man pages, so should we try to avoid it? 😄

@marchersimon I'd remove the comma in that suggestion though.

Comment (CleanMachine1) [reply pages/common/git-root.md:11]: Ok, after reading a few times I see your point, however I shouldn't of had to read it a few times to understand what it meant

Comment (bl-ue) [reply pages/common/git-root.md:11]: No...but not everyone does.  I, for one, knew what it mean with one glance. 🤷🏻‍♂️

Comment (CleanMachine1) [reply pages/common/git-root.md:11]: Well I obviously new what it does, since I wrote this :) However it just sounds a little confusing however if you agree with it then we will use it

Comment (bl-ue) [reply pages/common/git-root.md:11]: "relative to" is common and it's the most succinct way of putting it IMO; we can't make everything perfect.  I think it's fine.

**Review by bl-ue — APPROVED** [review]

**Review by SethFalco — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Comment by sbrl** [comment]
Remember to leave pull requests open for at least 24 hours minimum to give people a chance to comment, @CleanMachine1 :-)

**Comment by marchersimon** [comment]
Could we maybe integrate that into tldr-bot? Blocking merging within the first 24 hours and sending a reminder if a PR has enough approvals after 24 hours?

**Comment by bl-ue** [comment]
Sounds like a good idea.  Open an issue on https://github.com/tldr-pages/tldr-bot.  It'll be part of the backlog of changes we have, including a `/` command for `set-more-info-link.py`, notifying client maintainers of client spec updates via GitHub issues, etc., etc.

**Comment by CleanMachine1** [comment]
I think 24 is a little too much. Maybe 12

**Comment by bl-ue** [comment]
We're all in different timezones, so during your proposed _twelve_ hour wait, some of us might mostly be in bed.  24 hours ensures that regardless of timezone we've all had enough time to look about the PRs.

**Comment by marchersimon** [comment]
> I think 24 is a little too much. Maybe 12

I don't think so. In a week it doesn't matter if the page was merged 168 or 156 hours ago and to be save we should give anyone a chance to comment.

> We're all in different timezones, so during your proposed twelve hour wait, some of us might mostly be in bed. 24 hours ensures that regardless of timezone we've all had enough time to look about the PRs.

I even prefer 2 days, since we can't really expect anyone to check their GitHub messages every single day.

**Comment by SethFalco** [comment]
Just a question, isn't the whole point of approving that people are happy with it? 🤔 

Wouldn't it make more sense to:
* Make a rule where a pull request author shouldn't merge their pull request. 
* Increase the number of required approvals.
* Be more firm on when to approve (I've seen quite a few where there are pending comments, but another maintainer will approve the PR where there are unresolved discussions.)

Either one or multiple of the above. While I'm sure businesses would love to disagree 😆 , I don't think time should have part of if something should be merged or not?

**Comment by CleanMachine1** [comment]
Maybe 3 approvals and 12 hours? since most of the time here, 3 maintainers is all PR will get

**Comment by marchersimon** [comment]
I also don't think increasing the minimum amount of approvals is neccesary, since it's just a minimum.

Regarding the time, I think there is no reason to rush with merging PRs. Many PRs come at the same time a day and if we would merge within 12 hours, some wouldn't have a chance to comment on them. Also, you shouldn't be punished by not being able to review just because you don't check Github regularly. As I said, in some time in won't matter at all when the PR was merged. I would still prefer 2 days in most cases.

**Comment by CleanMachine1** [comment]
I'll be sticking to 24 hours from now. And enforcing it on others PRs. 
However if the change was as small as changing -f to --force then whatever

**Comment by bl-ue** [comment]
I love getting PRs merged speedily (yesterday we had one that was merged about 3 or 4 minutes after it was created!) but I think it would be good to let 1-2 days of waiting.  The reason for that is, I really don't like when PRs are merged without me getting a chance to approve 😢

**Comment by CleanMachine1** [comment]
Is there any way we can set a time for changes bigger than +10

**Comment by CleanMachine1** [comment]
+1 maybe not 10

**Comment by CleanMachine1** [comment]
https://github.com/marketplace/actions/wait-sleep is this of any use/importance?

**Comment by marchersimon** [comment]
I think any action should happen after a specified amount of time, regardless of activity, approvals etc. I opened an issue in [tldr-pages/tldr-bot](https://github.com/tldr-pages/tldr-bot/issues/) about that.

**Pull request merged**
